integration_tests/test_game.py

```diff
@@ -0,0 +1,106 @@
+import asyncio
+
+import pytest
+
+# All test coroutines will be treated as marked.
+pytestmark = pytest.mark.asyncio
+
+
+async def simulate_game(host, *guests, results=[]):
+    all_clients = [host] + list(guests)
+    game_id = await host.host_game()
+    await host.configure_joining_player(host.player_id, 1)
+    for guest in guests:
+        await guest.join_game(game_id)
+        await host.configure_joining_player(guest.player_id, 2)
+
+    await host.send_message({
+        "target": "game",
+        "command": "GameState",
+        "args": ["Launching"]
+    })
+    for client in all_clients:
+        await client.read_until(
+            lambda msg: (
+                msg.get("command") == "game_info" and
+                msg["host"] == "test" and
+                msg["launched_at"] is not None
+            )
+        )
+
+    await host.send_message({
+        "target": "game",
+        "command": "EnforceRating"
+    })
+
+    for client in all_clients:
+        for result in results:
```

Comment:
It doesn't matter right now and might not ever, but I guess the more "realistic" order of messages here might be
```python
for result in results:
  for client in all_clients:
    # ...
```
That is, I expect people to die one after the other and messages being sent in response to someone dying.
(Although it's true that individual clients rather send their messages in bursts at the end when there is a lot happening at once.)